src/main/java/com/naveenautomation/Base/TestBase.java

```diff
@@ -15,16 +15,11 @@
 public class TestBase {
 
 	
-
-/*
- * This is a base class
- * 
- */
-
-	public static WebDriver driver;
-	public Browsers DEFAULT_BROWSER = Browsers.GOOGLE_CHROME;
-	public static WebdriverEvents events = new WebdriverEvents();
-	public EventFiringWebDriver eventFiringWebDriver;
+	
+	  public static WebDriver driver; public Browsers DEFAULT_BROWSER =
```

Comment:
this is not correct, why u want to change it in this order

Reply:
I have changed the order now.